test/template.test.js

```diff
@@ -124,6 +124,8 @@ describe('template', () => {
         expect(render({'{{null}}': 'bar'})).toEqual({});
         expect(render({'{{""}}': 'bar'})).toEqual({});
         expect(render({'{{\'\'}}': 'bar'})).toEqual({});
+        expect(render({ "{{spreadValue()}}": { foo: "bar" } })).toEqual({foo: "bar"});
+        expect(render({ " {{ spreadValue() }} ": { foo: "bar" } })).toEqual({foo: "bar"});
```

Comment:
Can you add a test for object value with string jq?

Reply:
👌 added some